R/fit_single_contact_model.R

```diff
@@ -26,7 +26,7 @@ fit_single_contact_model <- function(contact_data, population) {
       contacts ~
       # multiplicative offset for population in the 'to' age group, to account for
       # opportunity to contact
-      stats::offset(log(pop_age_to)) +
```

Comment:
@njtierney this was the bug - I think because `offset()` is a special, so parsed in interpreting the formula, rather than the function being executed. Although there is a function in stats with this name (which does nothing), nammespacing it here trips up the formula interpretation, and causes it to be used as covariate.